manager/config/control-plane-security/authentication.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: "security.istio.io/v1beta1"
+# This policy enables mutual TLS traffic between every two workloads that have sidecar.
+kind: "PeerAuthentication"
+metadata:
+  name: "premissive-mtls-in-control-plane"
+  namespace: system
+spec:
+  mtls:
+    # PERMISSIVE: Workloads accept both mutual TLS and plain text traffic
+    mode: PERMISSIVE
```

Comment:
Don't we need another `PeerAuthentication` with `STRICT` mode targeting connectors workloads?

Reply:
We can however because the manager and other connectors have sidecars Istio auto mTLS feature guarantees that the mTLS is used for the connectors.

Reply:
It seems that PeerAuthentication  STRICT mode is not needed in our case because of Istio auto mTL feature which automatically enables mTLS between all workloads with sidecar (as in our case - the incoming services to the connectors have sidecar). STRICT mode is needed to prevent non mTLS traffic when not all services have sidecar.  For this case destination rule should also be used.

Reply:
You explanation is actually for why a destination rule is not needed. Destination rules are applied to the source (cf. destination) and can dictate whether an outgoing connection needs to be with mTLS. I agree that this is not needed here.

However, my understanding is that PeerAuthentication applies to the destination and set what it's willing to accept. Even if our sources are assumed to be configured to initial mTLS (via destination rules / default), the destination should still be set to only accept mTLS, even for just the case of misconfiguration.